pkgs/os-specific/linux/ndiswrapper/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv, fetchFromGitHub, kernel, perl, kmod, libelf }:
 let
-  version = "1.62-pre";
+  version = "1.63";
```

Comment:
Where does this version number come from? The latest tag on GitHub is v1.61 and the latest commit to the README is "Updated version to 1.61".

If this version is valid and there doesn't exist a tag on GitHub, please add a short comment to the version that reminds people to also update the `rev` of the source.

Reply:
I've checked that the code matches https://sourceforge.net/projects/ndiswrapper/files/stable/ v1.63 -- but really it would probably be better to switch the upstream to sf.net instead of github if the git repo maintainer isn't using tags consistently.

Reply:
I didn't realize that the GH version is a mirror. Is there any reason we aren't using the upstream version in the first place?

Reply:
Changed to use SourceForge.